components/game/game.js

```diff
@@ -69,10 +107,19 @@ const Game = () => {
     }, 1000);
   };
 
+
   /**
    * @description Changes the player turn to the next player.
    */
-  const changeTurn = () => setPlayerTurn(playerTurn === 3 ? 0 : playerTurn + 1);
+  const changeTurn = () => {
+    console.log('Changing Players Turn');
+    const nextTurn = playerTurn === 3 ? 0 : playerTurn + 1;
+    if (hands[nextTurn].hand.length === 0) {
+      setPlayerTurn(nextTurn + 1);
+    } else {
+      setPlayerTurn(nextTurn);
+    }
```

Comment:
hm, this can still be written as a ternary if you prefer it to be concise

```suggestion
    const nextTurn = playerTurn === 3 ? 0 : playerTurn + 1;
    const nextTurnCanPlay = hands[nextTurn].hand.length > 0;
    setPlayerTurn(nextTurnCanPlay ? nextTurn : nextTurn + 1);
```